### DISCUSSION TITLE
Trace Path

### DISCUSSION THREAD

**Discussion by denganjia** [Tencent/libpag#2414] [Q&A] [answered]
导出时提示我`Trace 形状图层 1: 路径 1 [9.1]: 暂不支持效果:"Pseudo/ADBE Trace Path".`
现在还不支持路径追踪吗

**Comment by kevingpqi123** [comment, marked as answer]
pag 支持的 AE 功能可以在官网中查找到：
https://pag.art/feature.html#ae

如果使用了不支持的 AE 特性，保底可以使用 BMP 预合成导出
https://pag.art/docs/ae-bmp-guide.html
